e2e-scripts/sippy-e2e-sippy-e2e-setup-commands.sh

```diff
@@ -233,9 +233,9 @@ ${KUBECTL_CMD} -n postgres describe job sippy-load-job
 # We set +e to avoid the script aborting before we can retrieve logs.
 set +e
 # This takes under 3 minutes so 5 minutes (300 seconds) should be plenty.
-TIMEOUT=300s
-echo "Waiting up to ${TIMEOUT} for the sippy-load-job to complete..."
-${KUBECTL_CMD} -n postgres wait --for=condition=complete job/sippy-load-job --timeout ${TIMEOUT}
+
+echo "Waiting up to ${SIPPY_LOAD_TIMEOUT:=300s} for the sippy-load-job to complete..."
+${KUBECTL_CMD} -n postgres wait --for=condition=complete job/sippy-load-job --timeout ${SIPPY_LOAD_TIMEOUT}
```

Comment:
Looks like we may log one thing and do another as the use of the var doesn't have the defaulting? Suggest actually setting the value with default and then just use the var in both of these lines.

Reply:
I used the := in that case to set SIPPY_LOAD_TIMEOUT per the link Dennis had shared https://unix.stackexchange.com/questions/594841/how-do-i-assign-a-value-to-a-bash-variable-if-that-variable-is-null-unassigned-f/594845

But can refactor for readability if you like

Reply:
Ah this actually *sets* the var? I had no idea, this is all good then, thanks.